### PR TITLE
BAH-3439 || Read also the timezone of birthtime.

### PR DESCRIPTION
**Pull request by harshkumar8789**
BAH-3439 || Change the timezone of birthtime.

### CODE REVIEW

**Comment by gsluthra** [comment]
A unit test related to Operation Theatre Surgical blocks is failing. Plz check @harshkumar8789 @rahu1ramesh

File: ui/app/common/util/dateUtil.js [hunk 1]
```diff
@@ -270,7 +270,7 @@ Bahmni.Common.Util.DateUtil = {
     },
 
     parseLongDateToServerFormat: function (longDate) {
```
Comment (gsluthra) [inline ui/app/common/util/dateUtil.js:272]: This seems to be a method in dateUtils.js (which is a common utility), and hence there will possibly be multiple users of this method. Have you verified that all callers of this method will continue to work fine with this change. Alternatively introduce a new method called, parseLongDateToServerFormatWithTimezone( ) so, only callers which care about this information call it.

Comment (harshkumar8789) [reply ui/app/common/util/dateUtil.js:272]: @gsluthra we did manual check with @gokultw . it seems that this changes doesnot effect, anywhere this method has been used.

Comment (gokultw) [reply ui/app/common/util/dateUtil.js:272]: we identified one issue in appointments but that is an existing issue.

**Review by gsluthra — APPROVED** [review]

File: ui/app/common/util/dateUtil.js (hunk 1 above)
Comment (rahu1ramesh) [reply ui/app/common/util/dateUtil.js:272]: @gokultw do we have a card created for the appointments issue?

Comment (gokultw) [reply ui/app/common/util/dateUtil.js:272]: I need to check with CURE team.

**Comment by gsluthra** [comment]
@akhilmalhotra : The PR name is confusing. It is basically saying, don't ignore the timezone information while reading birthtime, so that on UI the same time is shown later also, as was entered while registering the patient.

**Pull request merged**